Source/cmake/FindGLibCompileResources.cmake

```
@@ -0,0 +1,84 @@
+pkg_get_variable(GLIB2_PREFIX glib-2.0 prefix)
+find_program(
+    GLIB_COMPILE_RESOURCES_EXE
+    NAMES glib-compile-resources
+    HINTS ${GLIB2_PREFIX}
+    REQUIRED
+)
+execute_process(
+    COMMAND ${GLIB_COMPILE_RESOURCES_EXE} --version
+    OUTPUT_VARIABLE glib_compile_resources_version
+    OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+message(STATUS "Found glib-compile-resources (required): "
+    ${GLIB_COMPILE_RESOURCES_EXE}
+    " (${glib_compile_resources_version})")
+if (${glib_compile_resources_version} VERSION_LESS 2.77)
+    set(glib_compile_resources_has_depfile_bug TRUE)
+    message(STATUS "  Needs workaround for depfiles (will be used automatically)")
+else ()
+    set(glib_compile_resources_has_depfile_bug FALSE)
+    message(STATUS "  Doesn't need workarounds")
+endif ()
+
+function(GLIB_COMPILE_RESOURCES)
+    set(zeroArgKeywords "")
+    set(oneArgKeywords OUTPUT SOURCE_XML DEPFILE)
```

From what I can see there is no need to have a `DEPFILE` keyword argument. This function should instead add a dependency file automatically:

```suggestion
    set(oneArgKeywords OUTPUT SOURCE_XML)
```

The path of the dependency file can be derived from the output file path, see my suggestion below.